gnd/src/main/java/com/google/android/gnd/ui/common/photoview/PhotoView.java

```diff
@@ -0,0 +1,58 @@
+package com.google.android.gnd.ui.common.photoview;
+
+import android.content.Context;
+import android.net.Uri;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import androidx.fragment.app.Fragment;
+import com.google.android.gnd.databinding.PhotoViewBinding;
+import com.google.android.gnd.model.observation.Response;
+import com.google.android.gnd.system.StorageManager;
+import com.google.android.material.card.MaterialCardView;
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+
+public class PhotoView extends MaterialCardView {
+
+  private final PhotoViewBinding binding;
+
+  @Inject StorageManager storageManager;
+
+  @Nullable private Response response;
+
+  public PhotoView(Context context) {
+    this(context, null);
+  }
+
+  public PhotoView(Context context, AttributeSet attrs) {
+    this(context, attrs, 0);
+  }
+
+  public PhotoView(Context context, AttributeSet attrs, int defStyleAttr) {
+    super(context, attrs, defStyleAttr);
+
+    LayoutInflater inflater =
+        (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+    binding = PhotoViewBinding.inflate(inflater);
+  }
+
+  public void setResponse(@Nullable Response response) {
```

Comment:
Since Views are tightly bound to the Android OS and running VM, in general we're trying keep views as "dumb" as possible. This way we can test all/most business logic using ViewModels and upstream classes only. So ideally the views would delegate interaction with custom model classes or logic to the ViewModel. In most cases, that means the View can go away altogether, leaving us with just a layout and viewmodel. Would that be possible in this case?